src/components/Button.js

```diff
@@ -0,0 +1,42 @@
+const Button = () => (
+  <>
+    <header className="header">
+      <div className="calc">0</div>
+    </header>
+    <button className="btn" type="button">
+      Ac
+    </button>
+    <button type="button">+/-</button>
+    <button type="button">%</button>
+    <button type="button" className="color">
+      <span>÷</span>
+    </button>
+    <button type="button">7</button>
+    <button type="button">8</button>
+    <button type="button">9</button>
+    <button type="button" className="color">
+      <span>x</span>
+    </button>
+    <button type="button">4</button>
+    <button type="button">5</button>
+    <button type="button">6</button>
+    <button type="button" className="color">
+      -
+    </button>
+    <button type="button">1</button>
```

Comment:
Nice job so far, Nice job creating components for buttons👍🏼. Probably this was meant to be a child component that should be used in the calculator component but I noticed you are not making use of it anywhere😉.  But just as required, the calculator component should have a child component. Kindly break the calculator component into at least one child component. apart from this being a requirement, it is important for the separation of concerns and making our code base easily maintainable😉.


<img width="519" alt="image" src="https://github.com/Umuhan/calculator-app/assets/64643995/338f30ef-ea21-477c-8a32-242a6da630b0">